app/lib/describe_indexer.rb

```diff
@@ -39,20 +39,12 @@ def index
     end
   end
 
-  # Given a json document, return an XML string that contains
-  # the JSON blob as a CDATA element
+  # Converts the JSON payload to XML which is what Traject expects
   # @param [String] json
   # @return [String]
   def prep_for_indexing(json)
     parsed = JSON.parse(json)
-    xml = parsed.to_xml
-    doc = Nokogiri::XML(xml)
-    collection_node = doc.at('group')
-    cdata = Nokogiri::XML::CDATA.new(doc, json)
-    collection_node.add_next_sibling("<pdc_describe_json></pdc_describe_json>")
-    pdc_describe_json_node = doc.at('pdc_describe_json')
-    pdc_describe_json_node.add_child(cdata)
-    doc.to_s
+    parsed.to_xml
```

Comment:
This is the biggest change in this PR. Instead of adding a CDATA with the PDC record in the XML here...we create that XML in Traject (see below).